.github/workflows/ci.yml

```diff
@@ -82,8 +100,8 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: ["3.7"]
-        nautobot-version: ["1.3.4"]
+        python-version: ["3.8"]
+        nautobot-version: ["1.4"]
     env:
       INVOKE_NAUTOBOT_SSOT_ARISTACV_PYTHON_VER: "${{ matrix.python-version }}"
       INVOKE_NAUTOBOT_SSOT_ARISTACV_NAUTOBOT_VER: "${{ matrix.nautobot-version }}"
```

Comment:
Pylint shouldn't require a matrix at all: https://github.com/nautobot/nautobot/blob/develop/.github/workflows/ci_integration.yml#L119-L142

I think it's missing:
```
    env:
      INVOKE_NAUTOBOT_SSOT_ARISTACV_LOCAL: "True"
```

Reply:
When I specified the `INVOKE_NAUTOBOT_SSOT_ARISTACV_LOCAL` and the other envs the CI failed. I will touch base tomorrow with you @bryanculver. Thank you!